src/redux/actions/projects/updateProject.js

```diff
@@ -25,10 +25,9 @@ const updateProject = (
   const newProject = mergeObjectWithArrays(currentProject, diff);
 
   try {
+    // if config.currentApiVersion === api.V2 dont do any fetch, updating the experiment is enough
```

Comment:
do you mean that updating the project is enough? I see that we dispatch `PROJECTS_UPDATE` action type on line 33

Reply:
why did you comment out the if statement as well? It looks more clear to me if the if statement is in code